### ISSUE TITLE
Error in examples/linnear_mmv_hmdb.py

### ISSUE THREAD

**Issue by ALISURE** [deepmind/dmvr#1]
when runing to "for ex in examples:", I get the following error: 
```
tensorflow.python.framework.errors_impl.InvalidArgumentError: Name: <unknown>, Key: image/encoded, Index: 0.  Number of values != expected.  values size: 23 but output shape: []
	 [[{{node ParseSingleSequenceExample/ParseSequenceExample/ParseSequenceExampleV2}}]] [Op:IteratorGetNext]
```

Please help me solve this problem, thanks.

**Comment by AlyssaYoung** [comment]
Have you solved this problem? I am facing the same problem.

**Comment by lsmaira** [comment]
Thanks for filing this bug. Fixed in adf78b7a8edd4bb56c6ce1bffaa5003019761716
